src/modules/xiv/optimizer/optimizer.html

```diff
@@ -0,0 +1,43 @@
+<template>
+    <div class="container">
+        <h1>Rotation Optimizer</h1>
+        <form onsubmit={handleSubmit}>
```

Comment:
Technically, at least for accessibility, using a form is recommended, but I still find it weird in the SPA world